{
  "modified": "2025-08-28T07:10:40Z",
  "published": "2025-08-28T07:10:40Z",
  "schema_version": "1.5.0",
  "id": "MAL-2025-41625",
  "summary": "Malicious code in blabutt (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "blabutt",
        "purl": "pkg:pypi/blabutt"
      },
      "versions": [
        "99.99.99"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2025-04116",
        "import_time": "2025-08-29T06:41:38.627890578Z",
        "modified_time": "2025-08-28T07:10:40Z",
        "sha256": "9c909b833d41e12e551e25101458db08801c0f6b3a414e7518028433cd5aa96c",
        "source": "reversing-labs",
        "versions": [
          "99.99.99"
        ]
      }
    ]
  }
}
